core/src/consensus.rs

```diff
@@ -73,14 +83,20 @@ pub(crate) struct ComputedBankState {
 }
 
 #[frozen_abi(digest = "2ZUeCLMVQxmHYbeqMH7M97ifVSKoVErGvRHzyxcQRjgU")]
```

Comment:
Cool, #8012 ensures abi doesn't change even if we're adding new fields to this struct as long as they're `#[serde(skip)]`.